include/net/fota_download.h

```diff
@@ -135,6 +135,8 @@ int fota_download_init(fota_download_callback_t client_callback);
  * @param fragment_size Fragment size to be used for the download.
  *			If 0, @kconfig{CONFIG_DOWNLOAD_CLIENT_HTTP_FRAG_SIZE} is used.
  * @param expected_type Type of firmware file to be downloaded and installed.
+ * @param family Address family to be used for the download, AF_INET6 or AF_INET.
```

Comment:
should this be a new kconfig option instead of a function parameter?

Reply:
Well, this change is in response to someone changing the download_client library to set it at runtime instead of with a Kconfig option, so I was maintaining the same spirit.  Seems odd to simply move the goalpost.

Reply:
That's true, we do want to minimize that. We will see what other feedback comes.

Reply:
Agreed that minimizing API breaking changes should be avoided where feasible.